server/app/views/LoginForm.java

```diff
@@ -75,21 +75,6 @@ public LoginForm(
   }
 
   public Content render(Http.Request request, Messages messages, Optional<String> message) {
```

Comment:
The "new login page" never came to fruition, so remove the branched logic along with the `NEW_LOGIN_FORM_ENABLED` flag while we're here.